modules/neuroimaging/denoising/nlmeans/main.nf

```diff
@@ -0,0 +1,51 @@
+
+process DENOISING_NLMEANS {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.0.sif':
+        'scilus/scilus:2.0.0' }"
+
+    input:
+    tuple val(meta), path(image), path(mask)
+
+    output:
+    tuple val(meta), path("*_denoised.nii.gz")      , emit: image
+    path "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = []
+    if (mask) args += ["--mask $mask"]
+
+    """
+    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
+    export OMP_NUM_THREADS=1
+    export OPENBLAS_NUM_THREADS=1
+
+    scil_denoising_nlmeans.py $image ${prefix}_denoised.nii.gz 1 ${args.join(" ")}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        scilpy: 2.0.0
```

Comment:
Is there no way to get this version by running a command from the python script? (e.g. `scil_denoising_nlmeans.py --version`). this makes it easier to maintain when updating to a newer version. If that's not possible can you do something similar to this [module] (https://github.com/nf-core/modules/blob/e37348cc0271072ac6a3de88f87067ccb00de323/modules/nf-core/ale/main.nf) to make sure we don't forget to update the version